lumicks/pylake/fdcurve.py

```diff
@@ -15,7 +15,7 @@ class FdCurve(DownsampledFD):
     """An FD curve exported from Bluelake
 
     By default, the primary force and distance channels are `downsampled_force2`
-    and `distance1`. Alternatives can be selected using `FdCurve.with_channels()`.
+    and `distance1`. Alternatives can be selected using :func:`FdCurve.with_channels`.
```

Comment:
```suggestion
    and `distance1`. Alternatives can be selected using :func:`FdCurve.with_channels()`.
```

Reply:
Not necessary, as sphinx automatically adds the braces, if it is a function. But indeed nicer especially in the raw docstrings. -> fixed